playbooks/roles/edxapp/tasks/deploy.yml

```diff
@@ -354,19 +373,19 @@
 - name: update supervisor configuration
   shell:  "{{ supervisor_ctl }} -c {{ supervisor_cfg }} update"
   register: supervisor_update
-  sudo_user: "{{ supervisor_service_user }}"
+  become_user: "{{ supervisor_service_user }}"
   changed_when: supervisor_update.stdout is defined and supervisor_update.stdout != ""
   when: not disable_edx_services
   tags:
     - manage
 
 - name: ensure edxapp has started
-  supervisorctl: >
-    state=started
-    supervisorctl_path={{ supervisor_ctl }}
-    config={{ supervisor_cfg }}
-    name="edxapp:{{ item }}"
-  sudo_user: "{{ supervisor_service_user }}"
+  supervisorctl:
+    name: "edxapp:"
+    supervisorctl_path: "{{ supervisor_ctl }}"
+    config: "{{ supervisor_cfg }}"
+    state: started
+  become_user: "{{ supervisor_service_user }}"
   when: celery_worker is not defined and not disable_edx_services
   with_items: service_variants_enabled
```

Comment:
@itsjeyd Oops, you're absolutely correct.  Fixed in [35c11c1](https://github.com/hks-epod/configuration/commit/35c11c10fea3dc4bac9e2758dea4119cece0aa86).